README.md

```diff
@@ -0,0 +1,35 @@
+DDQ Backend Functional Tests
+============================
+
+Contains functional tests meant to be used by DDQ backend plugins. One example is [DDQ MySql Plugin](https://github.com/tests-always-included/ddq-backend-mysql).
+
+Usage
+-----
+
+Add this package as a devDependency and you can run the test script as part of your build process. You simply run the `run-test.sh` script. For instance with Travis and MySql.
+
+    services:
+        - mysql
+    before_script:
+        - bash script/mysql-commands.sh
+    script:
+        - bash node_modules/ddq-backend-tests/run-tests.sh
+
```

Comment:
You can borrow this newline from here if you like.  That is recycling the bytes on the computer.  ♲